doc/6/controllers/bulk/mWrite/index.md

```diff
@@ -59,9 +59,9 @@ Each created document is an object with the following properties:
 
 | Name      | Type              | Description  |
 | --------- | ----------------- | ------------------- |
-| \_id      | <pre>string</pre> | ID of the newly created document                       |
-| \_version | <pre>number</pre> | Version of the document in the persistent data storage |
-| \_source  | <pre>object</pre> | Created document                         
+| `\_id`      | <pre>string</pre> | ID of the newly created document                       |
```

Comment:
Shouldn't you get rid of that backslash now that these names are between backquotes?